package.json

```diff
@@ -76,6 +76,7 @@
     "@metamask/eslint-config-nodejs": "^9.0.0",
     "@metamask/eslint-config-typescript": "^9.0.1",
     "@types/jest": "^26.0.22",
+    "@types/jest-when": "^2.7.3",
```

Comment:
`jest-when` lets you limit assertions on mock functions to arguments. See the README for more: https://www.npmjs.com/package/jest-when